metricflow-semantics/tests_metricflow_semantics/snapshots/test_sg_resolver_output.py/str/test_complete_set_for_simple_metrics__result.txt

```diff
@@ -726,12 +728,6 @@ account_balance:
   │ user__verification__user__identity_verifications                           │ verification,user              │ METRIC           │ JOINED,METRIC                          │ accounts_source,id_verifications                               │
   │ user__view__user__views                                                    │ view,user                      │ METRIC           │ JOINED,METRIC                          │ accounts_source,views_source                                   │
   │ user__views                                                                │ user                           │ METRIC           │ JOINED,METRIC                          │ accounts_source,views_source                                   │
-  │ user__visit__user__visit_buy_conversion_rate                               │ visit,user                     │ METRIC           │ JOINED,METRIC                          │ accounts_source,buys_source,visits_source                      │
```

Comment:
do we know why all these are removed?

Reply:
See: https://github.com/dbt-labs/metricflow/pull/1882#discussion_r2408462110